code42.json

```diff
@@ -1065,6 +1065,280 @@
+                    "data_type": "string",
+                    "required": false,
+                    "primary": false,
+                    "default": "",
+                    "order": 4
+                },
+                "notes": {
+                    "description":  "Notes about user",
+                    "data_type": "string",
+                    "required": false,
+                    "primary": false,
+                    "default": "",
+                    "order": 5
+                }
+
+            },
+            "output": [
+                {
+                    "data_path": "action_result.parameter.username",
+                    "data_type": "string",
+                    "contains": [
+                        "email"
+                    ],
+                    "column_name": "Username",
+                    "column_order": 0
+                },
+                {
+                    "data_path": "action_result.parameter.org_uid",
+                    "data_type": "string",
+                    "column_name": "Org ID",
+                    "column_order": 1
+                },
+                {
+                    "data_path": "action_result.parameter.first_name",
+                    "data_type": "string",
+                    "column_name": "First Name",
+                    "column_order": 2
+                },
+                {
+                    "data_path": "action_result.parameter.last_name",
+                    "data_type": "string",
+                    "column_name": "Last Name",
+                    "column_order": 3
+                },
+                {
+                    "data_path": "action_result.parameter.notes",
+                    "data_type": "string",
+                    "column_name": "Notes",
+                    "column_order": 4
+                },
+                {
+                    "data_path": "action_result.data.userUid",
+                    "data_type": "string",
+                    "column_name": "User ID",
+                    "column_order": 5
+                },
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "column_name": "Status",
+                    "column_order": 6
+                },
+                {
+                    "data_path": "action_result.summary.user_id",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.summary.username",
+                    "data_type": "string"
+                }
+            ],
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "block user",
+            "identifier": "block_user",
+            "description": "Blocks a user from accessing their Code42 account",
+            "verbose": "Blocks a user from accessing their Code42 account (endpoint backup/security processes continue while blocked)",
```

Comment:
Verbose comments need to end in periods or validation gets mad
I noticed this when I made a change to an app in `phantom-apps`